sdk/cosmos/azure-cosmos/azure/cosmos/aio/database.py

```diff
@@ -144,40 +150,38 @@ async def read(self, **kwargs):
 
     @distributed_trace_async
     async def create_container(
-        self,
-        id,  # type: str  # pylint: disable=redefined-builtin
-        partition_key,  # type: Any
-        indexing_policy=None,  # type: Optional[Dict[str, Any]]
-        default_ttl=None,  # type: Optional[int]
-        offer_throughput=None,  # type: Optional[int]
-        unique_key_policy=None,  # type: Optional[Dict[str, Any]]
-        conflict_resolution_policy=None,  # type: Optional[Dict[str, Any]]
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> ContainerProxy
+            self,
+            id: str,  # pylint: disable=redefined-builtin
+            partition_key: PartitionKey,
```

Comment:
Curious why this isn't the `Union[str, int, bool, float]` like in other places?

Reply:
For creating the container you have to pass in a PartitionKey definition, which requires the PartitionKey object in order to specify a partition key path as well

The other methods were dealing with finding an item through a partition key value, at which point you only need to know what the item's partition key value is as opposed to the definition